index.js

```diff
@@ -0,0 +1,22 @@
+const enableModal = document.querySelector('#menu');
+const body = document.querySelector('body');
+const extBtn = document.getElementById('exit-btn');
+const modal = document.querySelector('#modal');
+const modalLinks = document.querySelector('.modal-nav-link');
+
+enableModal.addEventListener('click', () => {
+  modal.classList.remove('display-none');
+  body.classList.add('none-scroll');
+});
```

Comment:
- [ ] Kindly implement this suggested change by the previous reviewer because when I click on a mobile menu link and it takes me to the section, the page gets stuck. Kindly fix it 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/216179080-d536e0a9-e52c-4896-b52c-b84694158e52.png)

Reply:
@vickymarz Please use the code from the last commit for the review, I already fixed this issue

Reply:
@danielochuba I checked already, the last commit only contains changes to the MIT file

I tested the code, it has same issue